docs/12-ContinuousIntegration.md

```diff
@@ -119,8 +119,28 @@ Travis doesn't provide visualization for XML or HTML reports so you can't view r
 
 ## GitLab
 
-[Work In Progress]
+If a file ```.gitlab-ci.yml``` exists in the root of the git repository, gitlab will run a pipeline each time you push to the gitlab server. The file configures the docker image that will be called. Below is a sample which loads a php7 docker image, clones your files, installs composer dependencies, runs the built-in php webserver and finally runs codeception:
+
+```
+image: mychiara/php7
```

Comment:
The official php images don't have composer installed. That's why I choose another one that has it.
Of course you can use an official one like php:7.0-apache and install composer in it, but that will be less efficient and makes the documentation slightly longer and more difficult.